File: Business Logic Layer(Backend)/Reservation/Reservation.Core/Data/PaymentMethod.cs
using System;
using System.Collections.Generic;
namespace Reservation.Core.Data
{
    public partial class PaymentMethod
    {
        public PaymentMethod()
        {
            Payments = new HashSet<Payment>();
        }

        public decimal Payment_Method_Id { get; set; }
        public string Payment_Method_Name { get; set; }

        public virtual ICollection<Payment> Payments { get; set; }
    }
}

File: Presentation Layer (frontend)/Reservation_angular/src/data/paymentMethod.ts
import { Payment } from "./payment";

export class PaymentMethod {
    private payment_Method_Id: number=-1;
    private payment_Method_Name: string='';
  
public get Payment_Method_Id() : number {return this.payment_Method_Id; }
public set Payment_Method_Id(v :  number) {this.payment_Method_Id = v;}
public get Payment_Method_Name() : string {return this.payment_Method_Name; }
public set Payment_Method_Name(v :  string) {this.payment_Method_Name = v;}
/*payments: Payment[];
public get Payments() : Payment[] {return this.payments; }
public set Payments(v :  Payment[]) {this.payments = v;}
*/
}